scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -200,3 +200,35 @@ def test_from_dcm_ortho_output():
         eye3d[:, i, i] = 1.0
 
     assert_array_almost_equal(mult_result, eye3d)
+
+
+def test_from_1d_single_rotvec():
+    rotvec = [1, 0, 0]
+    expected_quat = np.array([0.4794255, 0, 0, 0.8775826])
+    result = Rotation.from_rotvec(rotvec)
+    assert_equal(result._single, True)
+    assert_array_almost_equal(result._quat[0], expected_quat)
+
+
+def test_from_2d_single_rotvec():
+    rotvec = [[1, 0, 0]]
+    expected_quat = np.array([[0.4794255, 0, 0, 0.8775826]])
+    result = Rotation.from_rotvec(rotvec)
+    assert_equal(result._single, False)
+    assert_array_almost_equal(result._quat, expected_quat)
+
+
+def test_from_generic_rotvec():
```

Comment:
Add some tests with small (but non zero) angles as well. Think how to test it. 

My ideas is that `w` should be close to 1 (up to quadratic term) and `x, y, z` should be approximately `0.5 * rotvec`. Also check that the norm is close to 1.